lib/note/index.js

```diff
@@ -292,23 +292,25 @@ const updateNote = async (req, res) => {
         title: Note.parseNoteTitle(content),
         content: content,
         lastchangeAt: now,
-        authorship: [
+        authorship: req.isAuthenticated() ? [
           [
             req.user.id,
```

Comment:
We keep authorship for the guest user, set `userId` to `null`:

```suggestion
            req.isAuthenticated() ? req.user.id : null,
```

Reply:
Hi, makes perfect sense - actually I had thought about updating the `authorship` somehow